src/main/java/com/blackducksoftware/integration/hub/item/HubItemList.java

```diff
@@ -0,0 +1,37 @@
+package com.blackducksoftware.integration.hub.item;
+
+import java.util.List;
+
+/**
+ * A list of items as returned by the Hub.
+ *
+ * @author sbillings
+ *
+ */
```

Comment:
it might be semantics but we could consider calling this HubItems but I'd avoid calling it a List unless it implemented java.util.List